addons/calendar_sms/security/sms_security.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="ir_rule_sms_template_system" model="ir.rule">
+        <field name="name">SMS Template: system administrator CRUD on calendar event templates</field>
+        <field name="model_id" ref="sms.model_sms_template"/>
+        <field name="groups" eval="[(4, ref('base.group_system'))]"/>
+        <field name="domain_force">[('model_id.model', '=', 'calendar.event'))]</field>
```

Comment:
Oops, there is a typo here ;-)

Reply:
Fuuuuu

Reply:
That was a copy paste from
https://github.com/odoo/odoo/blob/f332541bc84d6110c90d4c5e736892f185fff3df/addons/crm_sms/security/sms_security.xml#L7

fixed in #64924